### PR TITLE
Ready to plug into streams

### PR DESCRIPTION
**Pull request by TheoLeanse**
@gvonkoss @andygnewman

### CODE REVIEW

File: templates/main.html [hunk 1]
```diff
@@ -67,9 +65,13 @@
...
-    </li>
+			<a class="n-storylines__article-link" href="https://www.ft.com/content/{{id}}">{{title}}</a>
+			<p class="o-typography-timestamp">{{day}}</p>
+		</li>
 		{{/each}}
 	</ul>
 </div>
+<script type="text/javascript">
+	 window.FT = window.FT || {};
+	 window.FT.storylineData = {{{json this}}};
```
Comment (TheoLeanse) [inline templates/main.html:76]: this line will break the demos, because `json` is a helper from n-handlebars

Comment (TheoLeanse) [reply templates/main.html:76]: ...and I'm not sure how simple it will be to use n-handlbars here

Comment (TheoLeanse) [reply templates/main.html:76]: on the plus side it means we can develop it in the stream page!

Comment (TheoLeanse) [reply templates/main.html:76]: but I think it may be fixed now...

Comment (TheoLeanse) [reply templates/main.html:76]: yep ignore me

File: templates/main.html [hunk 2]
```diff
@@ -13,50 +13,48 @@
...
-						<p class="n-storylines__heatmap-segment-name o-typography-aside__body--small">{{ name}}</p>
+				<div class="n-storylines__heatmap-segment-colour n-storylines__all-months" style="opacity: {{weight}};"></div>
+				<p class="n-storylines__heatmap-segment-name o-typography-aside__body--small">{{ name}}</p>
 			</div>
-		{{/each}}
+			{{/each}}
 		</div>
 		{{else}}
-		<img class="n-storylines__back-btn" src="http://image.webservices.ft.com/v1/images/raw/fticon-v1:arrow-left?source=o-icons"/>
+			<img class="n-storylines__back-btn" src="http://image.webservices.ft.com/v1/images/raw/fticon-v1:arrow-left?source=o-icons"/>
```
Comment (andygnewman) [inline templates/main.html]: Should be using the new image service url https://www.ft.com/__origami/service/image/v2/images/raw/fticon-v1:arrow-left?source=o-icons

File: templates/main.html (hunk 1 above)
Comment (gvonkoss) [reply templates/main.html:76]: totally ignored. CHIP IT :shipit:

**Pull request merged**